app/windows/Details/window.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * This window will show some information and allow the user to tune the
+ * application's settings/preferences.
+ */
+
+import path from 'path'
+import url from 'url'
+
+import { BrowserWindow, remote } from 'electron'
+
+import isRenderer from 'is-electron-renderer'
+
+// Allow us to use create() in both electron windows and main process
+let BrowserWindowClass
+if (isRenderer) {
+  BrowserWindowClass = remote.BrowserWindow
+} else {
+  BrowserWindowClass = BrowserWindow
+}
+
+module.exports = {}
+
+module.exports.create = function createSettingsModal (app, hash) {
+  // Create the browser modal window.
+  let thisWindow = new BrowserWindowClass({
+    title: hash,
+    parent: app.mainWindow,
+    modal: true,
```

Comment:
Maybe: do not use it as a modal (up for discussion). See previous Screenshot on macOS